Server/src/index.ts

```diff
@@ -79,6 +79,12 @@ class CitrineOSServer {
             initSwagger(this._config, this._server);
         }
 
+        // Add Directus Message API flow creation if enabled
+        if (this._config.util.directus?.generateFlows) {
+            const directusUtil = new DirectusUtil(this._config, this._logger);
+            this._server.addHook("onRoute", directusUtil.addDirectusMessageApiFlowsFastifyRouteHook.bind(directusUtil));
+        }
```

Comment:
Sorry, should have put it in here as well right away. Had the below suggestion on the Swarm index.ts to log when then fastify server is ready and has also set up all the Directus flows

```suggestion
            this._server.addHook('onReady', async () => {
                this._logger.info('Directus actions initialization finished');
            });
        }
```